hackathon_site/dashboard/frontend/src/slices/hardware/cartSlice.ts

```diff
@@ -26,6 +30,47 @@ export const cartReducerName = "cart";
 export const initialState = cartAdapter.getInitialState(extraState);
 export type CartState = typeof initialState;
 
+// Thunk
+interface RejectValue {
+    status: number;
+    message: any;
+}
+
+export const submitOrder = createAsyncThunk<
+    APIListResponse<Hardware>,
+    { keepOld?: boolean } | undefined,
```

Comment:
Not needed here, this was just for fetching hardware to control of we wanted to replace everything in the store or not.